src/components/Dao/DaoContainer.tsx

```diff
@@ -98,6 +102,28 @@ class DaoContainer extends React.Component<IProps, IState> {
     this.setState({ memberDaos: memberDaos });
   }
 
+  public handleNewProposal = async (schemeId?: string): Promise<void> => {
+    const { showNotification, daoAvatarAddress } = this.props;
+
+    if (!await enableWalletProvider({ showNotification }, getNetworkByDAOAddress(daoAvatarAddress))) { return; }
+
+    if (typeof schemeId === "string") {
```

Comment:
is there a case where schemeId is not a "string". ?
can this condition be `if (schemeId)` ?

Reply:
Yes, there is a case, when schemeId is event. so we have to check, that type is exactly "string", or change the place, where we are using this function as event handler.

Reply:
Can you explain/give an example?

Reply:
[Here is the place where we are using this method as a event handler](https://github.com/daostack/alchemy/pull/2296/files/c460e9df9d144b1d8ed4889e329cb2ca89a847db#diff-8b2c028141c5d45e15a6a8b82d4edcdd55005613d480a0ce3ec57641b1237501R80)

[And this is a second place, where we are using this method as a function with string param](https://github.com/daostack/alchemy/pull/2296/files/c460e9df9d144b1d8ed4889e329cb2ca89a847db)

Reply:
are you sure the links above are correct ?

Reply:
@orenyodfat fixed second link. Right now both are correct. If you don't see hightlight, please, try to little bit scroll up and you will see hightlighted line.